catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/data/QueryTemplateMetacardImpl.java

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.catalog.ui.forms.data;
+
+import ddf.catalog.data.Metacard;
+import ddf.catalog.data.impl.types.CoreAttributes;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Metacard used for storing query templates in the catalog. Should not be used as a resource.
+ * Identifiable by the presence of {@link FormAttributes.Query#TAG} in {@link Metacard#TAGS}.
+ *
+ * <p>Relevant attributes:
+ *
+ * <ul>
+ *   <li>{@link Metacard#TITLE} - display name for a query template, in general not necessary, but
```

Comment:
Missing closing `</li>` tags

Reply:
I believe the formatter removes my closing tags despite my best efforts to add them.